lib/global-admin/addon/catalog/controller.js

```diff
@@ -25,7 +25,7 @@ export default Controller.extend({
         get(this, 'growl').fromError('Error removing Library', err);
       }).finally(() => {
         set(this, 'togglingLibrary', false);
-        this.send(this, 'refresh');
+        this.send('refresh', this);
```

Comment:
These should all just be `this.send('refresh');`, the action in the route doesn't take an argument.